.examples/discord_example.py

```diff
@@ -0,0 +1,20 @@
+# This example uses discord.py
+
+import discord
+from discord.ext import commands
+import asyncio
+
+# iDoggo modules
+from iDoggo import dog1
+from iDoggo import dog2
+import iDoggo
+
+bot = commands.Bot(command_prefix="!")
+
+@bot.commands(pass_context=True)
+async def dog1(ctx):
+     doggo = iDoggo.dog1()
```

Comment:
Can be simplified just to `await bot.say(iDoggo.dog1())`

Reply:
eh.

Reply:
await is easier then setting up an async function. Either way, this all looks good.

Reply:
eh, i rushed it, should be fine though

Reply:
Yeah - its good. You can certainly do ore than I can with discord.